heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -364,7 +364,10 @@ proto::system::Status* TMaster::RegisterStMgr(
     // First check to see if that other guy has timed out
     if (!stmgrs_[stmgr_id]->TimedOut()) {
       // we reject the new guy
-      LOG(ERROR) << "Another stmgr exists with the same id and it hasn't timed out" << std::endl;
+      LOG(ERROR) << "Another stmgr exists at "
```

Comment:
@congwang Yeah we read context for debugging. My concern is: people can change/remove the log "Got a register stmgr request from ..." without knowing this change will affect another log.